### PR TITLE
NC-news-2

### PR DESCRIPTION
**Pull request by tr-tobi**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  console.log(err);
+  next(err);
+});
```
Comment (Slyose) [inline app.js]: best to remove this middleware - as it's not actually doing anything at this point. We want to keep our PRs clean and uncluttered as possible!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+describe("/topic", () => {
+  test("GET:200 sends an array of the topics to the end point ", () => {
+    return req(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topic).toEqual(expect.any(Array));
+        expect(Object.keys(res.body.topic[0])).toEqual(
+          expect.arrayContaining([])
+        );
```
Comment (Slyose) [inline __tests__/app.test.js]: This needs finishing off, how can we test that the output looks the way we want it to? We'd need to be more specific than just an array

**Review by Slyose — COMMENTED** [review]
Nice work so far, a few things to tweak before submitting!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+afterAll(() => db.end());
+
+describe("/topic", () => {
+  test("GET:200 sends an array of the topics to the end point ", () => {
+    return req(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topic).toEqual(expect.any(Array));
+        expect(Object.keys(res.body.topic[0])).toEqual(
```
Comment (MRombi) [inline __tests__/app.test.js]: The first two assertion are not needed as with the last you are checking both, we can remove them 🙂

File: app.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getTopics } = require("./controllers/controller");
+
+const app = express();
+app.use(express.json());
```
Comment (MRombi) [inline app.js]: We should remove this line as it is not being used at the moment 🙂

**Review by MRombi — APPROVED** [review]
Good job on this ticket! Please, action on the feedback I left and merge, no need to resubmit 😄

**Pull request merged**